app/main.py

```diff
@@ -2,0 +1,15 @@
+
+def copy_file_from_command_cp(command: str) -> None:
+    command_parsed = command.split(" ")
+    source_file_name = command_parsed[1]
+    destination_file_name = command_parsed[2]
+
+    if source_file_name == destination_file_name:
+        print(f"File wasn't copied: source file '{source_file_name}' "
+              f"has the same name as the destination file.")
+    else:
+        with open(source_file_name, "r") as source_file, \
```

Comment:
Code Style: The variable names 'source_file' and 'destination_file' are descriptive and correct, which is good. However, it is a common convention to use 'src' and 'dst' as abbreviations for 'source' and 'destination', respectively. It would be better to follow this convention in this case.